docs/packaging.md

```diff
@@ -0,0 +1,53 @@
+---
+part: pixi
+title: Packaging pixi
+description: How to package pixi for distribution with another package manager?
+---
+
+## Packaging
+
+This is a guide for distribution maintainers wanting to package pixi for a different package manager.
+Users of pixi can ignore this page.
+
+### Building
+
+Pixi is written in rust and compiled using cargo, which are needed as compile-time dependencies.
+At runtime pixi needs no dependencies in other than the runtime it was compiled against (`libc`, ...).
+
+To build pixi run
+```shell
+cargo build --locked --profile dist
+```
+Instead of using the predefined `dist` profile, which is optimized for binary size, you can also pass other options to
+let cargo optimize the binary for other metrics.
+
+### Build-time Options
+
+Pixi provides some compile-time options, which can influence the build
+
+#### TLS
+
+By default, pixi is built with Rustls TLS implementation. You can compile pixi using the platform native TLS implementation
+using by adding `--no-default-features --feature native-tls` to the build command. Note that this might add additional
```

Comment:
```suggestion
using by adding `--no-default-features --feature native-tls` to the build command. Note that this will add additional
```

Reply:
I used might, as this is platform specific. As far as I know native tls implementations on windows and maybe macos are included in the platform and don't add a dependency.

Reply:
Ah yeah, that's true